### PR TITLE
Finish the spec companion pages

### PR DESCRIPTION
**Pull request by ingydotnet**

### CODE REVIEW

File: spec/changes.md [hunk 1]
```diff
@@ -0,0 +1,82 @@
+# YAML Specification Changes
+
+The current version of the YAML language is **1.2** and the current YAML
+specification version is **1.2.1**.
+
+## Changes in v1.2.1 (YYYY-MM-DD)
+
+* Host the YAML specification sources in a [public repository^]
```
Comment (Thom1729) [inline spec/changes.md:8]: A minor point, but these sentences should probably end in periods.

Comment (ingydotnet) [reply spec/changes.md]: done

File: spec/1.2.2/ext/team.md [hunk 2]
```diff
@@ -1,16 +1,22 @@
...
-Specification and the overall direction of the YAML Data Language.
+The YAML Language Development Team is responsible for the YAML Specification
+and the overall direction of the YAML Data Language.
 
-The current team consists of:
+To see all the individual contributors for a specific YAML specification
+version, visit <https://github.com/yaml/yaml-spec/releases/>.
+
+
+## Team for spec version 1.2.1
```
Comment (Thom1729) [inline spec/1.2.2/ext/team.md:10]: 1.2.2

File: spec/1.2.2/ext/community.md [hunk 3]
```diff
@@ -15,16 +28,22 @@
...
   YAML developers hang out and field user queries from time to time.
 
+* [Twitter YAML News](https://twitter.com/yamlnews)
+
+  Follow us here to keep on the latest happenings in the life of YAML.
+
 * [Mailing List](https://sourceforge.net/projects/yaml/lists/yaml-core)
 
-  YAML was originally designed by its creators over this mailing list.
+  YAML was originally designed by its creators almost entirely on this mailing
```
Comment (Thom1729) [inline spec/1.2.2/ext/community.md:42]: Is this list active?

Comment (ingydotnet) [reply spec/1.2.2/ext/community.md:42]: Active in the sense that it works and we post major announcements there still, and from time to time someone asks a question there and we point them to better places to ask.

File: spec/1.2.2/ext/team.md (hunk 2 above)
Comment (ingydotnet) [reply spec/1.2.2/ext/team.md]: fixed in 4cf4b45

**Review by perlpunk — APPROVED** [review]

**Pull request merged**